app/javascript/widget/components/TeamAvailability.vue

```diff
@@ -72,11 +79,7 @@ export default {
     <div class="flex items-center justify-between gap-2">
       <div class="flex flex-col gap-1">
         <div class="font-medium text-n-slate-12">
-          {{
-            isOnline
```

Comment:
Let's specify that users should use a maximum number of characters to avoid text overflow. Let's add this to the documentation for now and implement truncation:

![CleanShot 2025-07-08 at 15 41 52](https://github.com/user-attachments/assets/cf370394-d4dd-4edc-9632-0735ae030eb6)

Reply:
Updated the PR. Now it will truncate the following fields based on line count:

* **`welcomeTitle`**: truncated if it exceeds **4 lines**
* **`welcomeDescription`**: truncated if it exceeds **6 lines**
* **`availableMessage/unavailableMessage`**: truncated if it exceeds **2 lines**